ghcide/src/Development/IDE/Main.hs

```diff
@@ -116,6 +117,7 @@ stderrLogger = do
 
 defaultMain :: Arguments -> IO ()
 defaultMain Arguments{..} = do
+    setLocaleEncoding utf8
```

Comment:
@jneira In `defaultMain` we've already set the encoding of `stdout` and `stderr` to UTF-8; plus `setLocaleEncoding utf8`, I think we could remove `withUtf8` from here:
https://github.com/haskell/haskell-language-server/blob/5724343ee2f861cba26fe430ff6413ff5f2f9909/exe/Main.hs#L13

and get rid of this dependency.

Reply:
The package has some utility functions but it seems `withUft8` does esentially that (also for stdin?)

https://github.com/serokell/haskell-with-utf8/blob/b60991303f9cf3f2891479c581549c2d54f45a0e/lib/Main/Utf8.hs#L78-L82

If we are not going to use its utility functions we can take ride of, yeah

Reply:
No worry, `stdin` is used in lsp mode, where [`runServerWithHandles`](https://hackage.haskell.org/package/lsp-1.1.0.0/docs/src/Language.LSP.Server.Control.html#runServerWithHandles) will set it to utf8